src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.Android.cs

```diff
@@ -241,6 +242,22 @@ private void UpdateTextFormatted()
 			_textFormatted = GetTextFormatted();
 		}
 
+		/// <summary>
+		/// Sets ellipsize to truncateAt.END when a number of line is specified.
+		/// If ellipsize is another value, the text won't truncate.
+		/// </summary>
+		private void UpdateMaxLines()
+		{
+			if (_ellipsize != null)
```

Comment:
You should probably reset `_ellipsize` to `null` when `MaxLines` changes (similar to `TextTrimming`)

Reply:
That is what I do, if it's not null, I return it, like in text trimming, if the layout is constrained, I set it to TruncateEnd, else I set it to null. What am I missing?

Reply:
look at what happens when the `TextTrimming` property changes

Reply:
I fixed this yesterday.